pkgs/by-name/ma/mago/package.nix

```diff
@@ -0,0 +1,38 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  testers,
+  mago,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "mago";
+  version = "0.0.5";
+
+  src = fetchFromGitHub {
+    owner = "carthage-software";
+    repo = pname;
+    rev = version;
```

Comment:
```suggestion
    tag = version;
```

Reply:
Are you sure about that ?

Reply:
Yep, see https://github.com/NixOS/nixpkgs/pull/355973

Reply:
Thanks for the link :+1:

Reply:
done